app/javascript/api/authApiService.ts

```diff
@@ -42,3 +42,15 @@ export const updatePassword = async (
     passwordConfirmation: passwordConfirmation,
     token: token,
   }).then(({ data }) => data.accessToken)
+
+export const updateNameOrDOB = async (user: User): Promise<User> => {
```

Comment:
nit: It looks email can be updated in this function too? If so, consider renaming the function to reflect that. `updateUserProfile`?

Reply:
Updating the email will need to be done in a different function as the endpoint for updating email is different (https://docs.google.com/document/d/18gurkqTOat8X55NHTFOoGsE16JqSqji71XwPTlcYlNI/edit#heading=h.t9yx58fn9p4l)

Reply:
Ah okay, so will the email field in this function be removed after that? Or is the email required to be sent alongside the name and DOB?

Reply:
Based on my testing I don't think that the email field is required, but it is included in the AngularJS version of this call so I included it as well. I figured it would be worthwhile to include just in case it is needed somewhere that I didn't see